curvefs/src/volume/common.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Date: Tuesday Jul 12 11:04:47 CST 2022
+ * Author: wuhanqing
+ */
+
+#include "curvefs/src/volume/common.h"
+
+#include <ostream>
+#include <string>
+
+#include "curvefs/src/volume/utils.h"
+
+namespace curvefs {
+namespace volume {
+
+std::ostream& operator<<(std::ostream& os, const AllocateHint& hint) {
```

Comment:
Implement this in AllocateHint maybe more clear.

Reply:
it's not a core function, so I think it's ok to put it here. And, put so many logic in header file will slow down compilation and linking.